src/main/java/de/samply/share/client/quality/report/file/excel/instances/basic/BasicExcelRowElements.java

```diff
@@ -13,9 +13,9 @@
   private final List<BasicExcelColumnMetaInfo> metaInfos;
 
   /**
-   * Todo.
+   * Contains information (title and eventually link) of each cell of an excel row.
```

Comment:
This comment is more suitable for the Class JavaDoc. On a constructor the comment should start with "Creates a ...".

Reply:
My descriptions of the constructors provide information about the class. But thanks for your advice. I will modify the java docs of the constructors.